_posts/2021-01-05-git-rebase-merge.md

```diff
@@ -0,0 +1,202 @@
+---
+
+title: "Git Rebase Vs Git Merge: Which One Is Better?"
+categories: [craft]
+modified: 2021-01-05
+excerpt: "If you are new to Git and/or GitHub, it's easy to get overwhelmed by the different workflow models you can use. 
+          This post explains the basic fork and pull workflow model that is used on a lot of GitHub repositories." 
+image:
+  auto: 0050-git
+---

```

Comment:
Thanks for sharing the details.
I have updated the author and date field in the article while adding my details and picture in the authors.yml and authors folder.
Yes, the preview is probably not loading due to that since it was loading locally for me when I checked it to preview.